test/e2e/lib/pages/stats-page.js

```diff
@@ -16,6 +16,14 @@ export default class StatsPage extends AsyncBaseContainer {
 		super( driver, By.css( '.stats-module' ) );
 	}
 
+	async openInsights() {
+		await driverHelper.clickWhenClickable( this.driver, By.css( '.stats-navigation__insights' ) );
```

Comment:
This test is failing on [mobile](https://circleci.com/gh/Automattic/wp-e2e-tests-for-branches/45030#tests/containers/0) because  _Insights_ button is hidden in the dropdown menu:

<img width="397" alt="Screenshot 2020-01-28 at 13 20 52" src="https://user-images.githubusercontent.com/7116222/73263851-c5c1bc80-41d1-11ea-86ec-981a3d28d370.png">

Adding screen check before this line should work for mobile screens:

```
if ( driverManager.currentScreenSize() === 'mobile' ) {
	await driverHelper.clickWhenClickable( this.driver, By.css( '.gridicons-chevron-down' ) );
}
```

Reply:
Before reading your comment, I reused `_expandNavIfMobile` inside of `pages/stats-page.js` to achieve the same goal. Is that ok, or would you prefer if I switched over to your approach?

Reply:
`_expandNavIfMobile` is perfect 👌 I didn't see it and it was there and so obvious :)

Reply:
Easy to miss, because it's in another file than the spec 🙂